README.md

```diff
@@ -27,6 +27,69 @@ Using rebar3:
 
     # rebar3 compile
 
+PER-APPLICATIONS CONFIG
+-----------------------
+
+erGW-AAA support per-application config.
+
+Example of possible config.
+
+```erlang
+ {ergw_aaa, [
+    {applications, [
+        {default,
+            {provider, ergw_aaa_radius, [
+                {nas_identifier, <<"nas_id1">>},
+                {radius_auth_server, {{127,0,0,1}, 1812, <<"secret">>}},
+                {radius_acct_server, {{127,0,0,1}, 1813, <<"secret">>}}
+            ]}
+        },
+        {application1,
+            {provider, ergw_aaa_radius, [
+                {nas_identifier, <<"nas_id3">>},
+                {radius_auth_server, {{127,0,0,1}, 1812, <<"radius_password">>}},
+                {radius_acct_server, {{127,0,0,1}, 1813, <<"radius_password">>}}
+            ]},
+            {attribute_map, [
+                {'Attr-A', ['Fixed-Value', '/', 'SomeElse']},
+                {'Attr-B', ['MAC']},
+                {'Attr-C', disabled}  %% do not transmit Attr-C
+            ]}
+        },
+        {application,
+            {provider, ergw_aaa_mock,
+                [{shared_secret, <<"MySecret">>}]
+            },
+            {attribute_map, [
+                %% send Called-Station-Id as "12-34-56-78-9A-BC (MyWifi)"
+                {'Called-Station-Id', ['BSSID', <<" (">>, 'SSID', <<")">>]},
+                {'Calling-Station-Id', ['MAC']},
+                {'TP-Location-Id', ['ZONE-X']}
+            ]}
+        }
+    ]}
+ ]},
+
+```
+
+Set 'AAA-Application-Id' key for select application config.
+Default ApplicationId is 'default'.
+
+FLEXIBLE AAA ATTRIBUTES
+-----------------------
+
+erGW-AAA have ability for dynamic attributes. You can set 'attribute\_map' for
+specific application. For example:
+
+```erlang
+{attribute_map, [
+    {'Attr-A', ['Fixed-Value', '/', 'SomeElse']},
+    {'Attr-B', ['MAC']},
+    {'Attr-C', disabled}  %% do not transmit Attr-C
+]}
+```
+
+If somthing variable in mapping rule not founed then replacesed on named variable.
```

Comment:
small spelling mistakes: "something", "not found", "replaced". 
please rephrase that sentence, it reads kind of "bumpy"